src/DynamoCoreWpf/ViewModels/Core/PortViewModel.cs

```diff
@@ -502,8 +502,15 @@ private void AutoCompleteCluster(object parameter)
 
             if (wsViewModel.DynamoViewModel.IsDNAClusterPlacementEnabled)
             {
-               MLNodeClusterAutoCompletionResponse results = wsViewModel.NodeAutoCompleteSearchViewModel.GetMLNodeClusterAutocompleteResults();
-               wsViewModel.OnRequestNodeAutoCompleteViewExtension(results);
+                try
+                {
+                    MLNodeClusterAutoCompletionResponse results = wsViewModel.NodeAutoCompleteSearchViewModel.GetMLNodeClusterAutocompleteResults();
+                    wsViewModel.OnRequestNodeAutoCompleteViewExtension(results);
+                }
+                catch (Exception e)
+                {
+                    // Log the exception and show a notification to the user
```

Comment:
Log the exception?

Reply:
Yes, I added this to handle a crash that I encountered while testing, I think there will be more work done for cluster autocomplete integration and this will likely change, so we can handle it then.